source/guide_jingo.rst

```diff
@@ -0,0 +1,257 @@
+.. highlight:: console
+
+.. sidebar:: About
+
+  This guide has been tested with Uberspace 7.1.1 and Jingo 1.8.5
```

Comment:
Could you move that to the bottom? We updated [the rules](https://github.com/Uberspace/lab/blob/master/source/README.md). Sorry, we're still experimenting :)

Reply:
ok did so, whats the best way to keep track of rule changes like this? Maybe you can set up an issue that can be subscribed?

Reply:
Will do.